src/psyclone/tests/psyir/nodes/node_test.py

```diff
@@ -470,7 +475,6 @@ def test_dag_names():
     psy = PSyFactory("dynamo0.3", distributed_memory=True).create(invoke_info)
     invoke = psy.invokes.invoke_list[0]
     schedule = invoke.schedule
-    assert super(Schedule, schedule).dag_name == "node_0"
```

Comment:
I added it back (now with a _1 since it has a container on top). The reason I removed it is because this node is not part of a Schedule and therefore can not be part of a DAG (so the name is a bit meaningless). But I guess one more assert doesn't hurt.